lib/api/api-fetch/util.js

```diff
@@ -181,7 +114,74 @@ function isValidHTTPToken (characters) {
   return true
 }
 
+function toWebReadable (streamReadable) {
```

Comment:
we should really have this in core.

Reply:
It is in core, just not released as far as I can tell.

Reply:
It's either:

`newStreamReadableFromReadableStream` or `newReadableStreamFromStreamReadable`.

I never remember which on does what.